nixos/modules/misc/ids.nix

```diff
@@ -339,6 +339,7 @@
       rss2email = 312;
       cockroachdb = 313;
       zoneminder = 314;
+      mailman = 316;
```

Comment:
No need to use static ids, see https://github.com/NixOS/nixpkgs/pull/65698

Reply:
I am sympathetic to the idea of removing the uid/gid assignment. 

I am about to re-target this PR to `master`, though, and the new uid/gid is already present there, so would have to remove them in a new PR.